elastalert/config.py

```diff
@@ -233,13 +234,14 @@ def load_modules(rule, args=None):
     if type(rule['alert']) != list:
         rule['alert'] = [rule['alert']]
     for alert in rule['alert']:
-        if alert in alerts_mapping:
+        if type(alert) == str and alert in alerts_mapping:
             rule_alerts.append(alerts_mapping[alert])
+        elif type(alert) == dict and alert.keys()[0] in alerts_mapping:
+            rule_alerts.append(alerts_mapping[alert.keys()[0]])
```

Comment:
Along with the comment below, I think this should use a separate list for inline alerts. You can create a dictionary mapping because there could be multiple of the same type, IE, {Email: {...}, Email: {...}} wouldn't work because the alert type keys would conflict. Instead, we can create tuples.
```
rule_type = alerts_mapping[alert.keys()[0]]
rule_config = alert.values()[0]
inline_alerts.append((rule_type, rule_config))
```
We should probably also assert that len(alert) == 1, because we are doing direct indexing.